pco_cli/src/dtypes.rs

```diff
@@ -143,6 +144,49 @@ parquetable!(f64, parquet::data_type::DoubleType, "DOUBLE");
 parquetable!(i32, parquet::data_type::Int32Type, "INT32");
 parquetable!(i64, parquet::data_type::Int64Type, "INT64");
 
+impl Parquetable for f16 {
```

Comment:
Not sure what we wonna do here.

Reply:
Let's just leave it as a TODO for now. This is only used for benchmarks, and we can add support in later.

Can you run `cargo clippy`? I think you'll get some warnings that variables are unused here and should be prefixed with `_`.